Content.Server/MachineLinking/System/ReleaseSignallerSystem.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Content.Server.DoAfter;
+using Content.Server.MachineLinking.Components;
+using Content.Server.Popups;
+using Content.Shared.Interaction.Events;
+using Content.Shared.MobState;
+using Content.Shared.Popups;
+using Content.Shared.Verbs;
+using Robust.Shared.Player;
+
+namespace Content.Server.MachineLinking.System;
+
+/// <summary>
+/// Entity system for release signallers, aka dead man's switches.
+/// </summary>
+public sealed class ReleaseSignallerSystem : EntitySystem
+{
+    [Dependency] private readonly SignalLinkerSystem _signalSystem = default!;
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly DoAfterSystem _doAfterSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ReleaseSignallerComponent, ComponentInit>(OnInit);
+        SubscribeLocalEvent<ReleaseSignallerComponent, DroppedEvent>(OnDropped);
+        SubscribeLocalEvent<ReleaseSignallerComponent, UseInHandEvent>(OnUsedInHand);
+        SubscribeLocalEvent<ReleaseSignallerComponent, GetVerbsEvent<AlternativeVerb>>(OnGetVerbs);
+        SubscribeLocalEvent<ReleaseSignallerComponent, ReleaseSignallerDisarmCompleteEvent>(OnSuccessfulDisarm);
+
+        SubscribeLocalEvent<ReleaseSignallerHolderComponent, MobStateChangedEvent>(OnMobStateChanged);
+    }
+
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+        foreach (var holder in EntityQuery<ReleaseSignallerHolderComponent>())
+        {
+            if (holder.Switches.Count == 0)
+            {
+                RemComp<ReleaseSignallerHolderComponent>(holder.Owner);
+            }
+        }
+    }
+
+    private void OnInit(EntityUid uid, ReleaseSignallerComponent component, ComponentInit args)
+    {
+        _signalSystem.EnsureTransmitterPorts(uid, component.Port);
+    }
```

Comment:
This is why tests fail, you need to put the component on the prototype. See https://discord.com/channels/310555209753690112/928056457117696020/1009262545888083988